### DISCUSSION TITLE
update icon inside the applicatio

### DISCUSSION THREAD

**Discussion by amirmohammadraei** [tutao/tutanota#6899] [Ideas]
Dear Development Team,

I've noticed that each time an update notification appears in my desktop application, I have to download it from Tuta's main website. Would it be possible to integrate the update functionality directly into the application? For example, could we add an icon for checking updates, so that when clicked, it automatically starts the download process? After the update is completed, the application could restart to reflect the new version.

Thank you for your consideration.

**Comment by charlag** [comment]
Hi, there is auto-update function on all desktop platforms. Clicking on notification should trigger it, as well as click on the message on the lock screen or going to settings -> desktop and updating from there.
